config_types.h

```diff
@@ -93,4 +93,20 @@ struct server_addr {
     int m_last_error;
 };
 
+struct config_cpu_list {
+    std::vector<unsigned int> cpu_list;
+    std::vector<unsigned int>::iterator next_cpu;
+
+    config_cpu_list() {}
+    explicit config_cpu_list(unsigned int cores);
+    explicit config_cpu_list(std::string str);
+    config_cpu_list(const config_cpu_list& copy);
+    config_cpu_list& operator=(const config_cpu_list& cl);
+
+    bool is_defined();
+    bool is_legal();
+    unsigned int get_next_cpu();
+    const char *print(char* buf, unsigned int size);
+};
```

Comment:
Why not make these four methods `const`?

Reply:
is_defined(), is_legal(), print(char* buf, unsigned int size) should be const, changed